Dockerfile

```diff
@@ -1,28 +1,22 @@
-FROM maven:3.8-eclipse-temurin-21-alpine as builder
+FROM maven:3.8-eclipse-temurin-21-alpine AS builder
 
 WORKDIR /home
 
-# Copy only the files needed to avoid putting all sorts of junk from your local env on to the image
-COPY ./jpo-ode/pom.xml ./jpo-ode/
-COPY ./jpo-ode/jpo-ode-common/pom.xml ./jpo-ode/jpo-ode-common/
-COPY ./jpo-ode/jpo-ode-common/src ./jpo-ode/jpo-ode-common/src
-COPY ./jpo-ode/jpo-ode-plugins/pom.xml ./jpo-ode/jpo-ode-plugins/
-COPY ./jpo-ode/jpo-ode-plugins/src ./jpo-ode/jpo-ode-plugins/src
-COPY ./jpo-ode/jpo-ode-core/pom.xml ./jpo-ode/jpo-ode-core/
-COPY ./jpo-ode/jpo-ode-core/src ./jpo-ode/jpo-ode-core/src/
-COPY ./jpo-ode/jpo-ode-svcs/pom.xml ./jpo-ode/jpo-ode-svcs/
-COPY ./jpo-ode/jpo-ode-svcs/src ./jpo-ode/jpo-ode-svcs/src
+ARG MAVEN_GITHUB_TOKEN_NAME
+ARG MAVEN_GITHUB_TOKEN
```

Comment:
This automated warning has a good point. This should be fixed.  It would not be good to have user's tokens exposed in the docker images.

"GitHub Actions
/ jpo-geojsonconverter

Sensitive data should not be used in the ARG or ENV commands

SecretsUsedInArgOrEnv: Do not use ARG or ENV instructions for sensitive data (ARG "MAVEN_GITHUB_TOKEN")
More info: https://docs.docker.com/go/dockerfile/rule/secrets-used-in-arg-or-env/"

Reply:
Interestingly enough, I don't think the final image actually ends up with the tokens exposed. One of the builder images needs the tokens, but the final image only copies in the compiled jar file. If you open the final container it doesn't have the environment variable embedded in it.

Reply:
I think you are right.  I too don't see the token among the environment variables in the final runtime image.  Also Docker's documentation on scoping here is reassuring; it says that ENV variables in Dockerfiles can't be globally scoped.  ie. if ENV appears after the FROM statement, it is locally scoped to only the first stage.
https://docs.docker.com/build/building/variables/#scoping